dev/com.ibm.ws.security.openidconnect.server/resources/com/ibm/ws/security/openidconnect/server/internal/resources/OidcServerMessages.nlsprops

```diff
@@ -192,9 +192,9 @@ LOGOUT_TOKEN_ERROR_GETTING_CLAIMS_FROM_ID_TOKEN=CWWKS1643E: The OpenID Connect p
 LOGOUT_TOKEN_ERROR_GETTING_CLAIMS_FROM_ID_TOKEN.explanation=The ID token might be malformed or a claim within the token might have a value that is an unexpected data type. Alternatively, the token might be missing claims that are required or the issuer of the token is not the issuer that is expected.
 LOGOUT_TOKEN_ERROR_GETTING_CLAIMS_FROM_ID_TOKEN.useraction=Ensure the ID token is formatted correctly. See the error that is included in the message for more information.
 
-LOGOUT_TOKEN_ERROR_GETTING_CLIENTS_TO_LOG_OUT=CWWKS1644E: The OpenID Connect provider cannot determine the set of OpenID Connect clients to log out because of the following error: {0}
-LOGOUT_TOKEN_ERROR_GETTING_CLIENTS_TO_LOG_OUT.explanation=The provider failed to retrieve the set of clients that are registered, or the ID token claims might be malformed.
-LOGOUT_TOKEN_ERROR_GETTING_CLIENTS_TO_LOG_OUT.useraction=For more information, see the error included in the message.
+ERROR_RETRIEVING_CLIENT_TO_BUILD_LOGOUT_TOKENS=CWWKS1644W: The {0} OpenID Connect provider encountered the following error when it retrieved the {1} client to build back-channel logout tokens: {2}
```

Comment:
**NOTE:** CWWKS1644W being a warning message, I don't think we can say `following error'

Reviewed CWWKS1644W: 
The {0} OpenID Connect provider displays the following message when it retrieved the {1} client to build back-channel logout tokens: {2}

Reply:
I think the suggested change is more clunky and more difficult to translate than the message as it is. ID style in the past has always seemed to steer towards, "The &lt;thing doing the processing> encountered an error/the following error while &lt;doing something>: &lt;the error>". While the suggestion is valid English, it's a more confusing structure to the sentence.